dspace-api/pom.xml

```diff
@@ -772,6 +772,12 @@
             <version>2.6.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.javax.persistence</groupId>
+            <artifactId>hibernate-jpa-2.1-api</artifactId>
+            <version>1.0.0.Final</version>
+            <scope>compile</scope>
```

Comment:
please remove the `<scope>`, as it's not needed.  "compile" is the default, and all our other dependencies leave this scope out.